src/v2/spec.yaml

```diff
@@ -3973,6 +3983,9 @@ tags:
     description: Operations related to a ban
 
 paths:
+  /{path}:
```

Comment:
Am not sure if this will add the headers to all the paths.
Checked in swagger, params are empty

<img width="747" alt="image" src="https://github.com/user-attachments/assets/06950d3e-e0b7-4594-ac69-ce476967ddb9">

We'll have to add parameters individually in each path

<img width="747" alt="image" src="https://github.com/user-attachments/assets/fec79d17-0aa1-4029-82e3-8a7badb20534">